### ISSUE TITLE
How to track details of an associated model?

### ISSUE THREAD

**Issue by xhnagata** [cowbell/active_model-errors_details#5]
``` ruby
class Author < ActiveRecord::Base
  has_many :books
  accepts_nested_attributes_for :books

  validates :name, presence: true
end

class Book < ActiveRecord::Base
  belongs_to :author

  validates :title, presence: true
end
```

``` ruby
> author = Author.new
> author.books.build
> author.valid?
> author.errors.details
=> {:name=>[{:error=>:blank}]}
> author.errors.messages
=> {:"books.title"=>["can't be blank"], :name=>["can't be blank"]}
```

I want to take errors of a book model from `author.errors.details` like `author.errors.messages`. 
Is there a way?

**Comment by morgoth** [comment]
@xendoc I'll take a look at rails internals, as this probably also won't work in upcoming 5.0. Thanks

**Comment by xhnagata** [comment]
@morgoth Thanks for your response.
I hope to be improved!

**Comment by morgoth** [comment]
I submitted PR to rails https://github.com/rails/rails/pull/22105 - when it will be accepted, I'll try to monkey patch rails prior to 5.0 to support this.

**Comment by morgoth** [comment]
@xendoc It's now fixed in 1.3.0 version

**Comment by xhnagata** [comment]
Thank you! :smile:
